core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -316,20 +321,24 @@ public String toString() {
   public static final PropertyKey UNDERFS_OSS_CONNECT_MAX =
       new Builder(Name.UNDERFS_OSS_CONNECT_MAX)
           .setDefaultValue(1024)
+          .setDescription("The maximum number of OSS connections")
           .build();
   public static final PropertyKey UNDERFS_OSS_CONNECT_TIMEOUT =
       new Builder(Name.UNDERFS_OSS_CONNECT_TIMEOUT)
           .setAlias(new String[]{"alluxio.underfs.oss.connection.timeout.ms"})
           .setDefaultValue("50sec")
+          .setDescription("The timeout when connecting to OSS")
           .build();
   public static final PropertyKey UNDERFS_OSS_CONNECT_TTL =
       new Builder(Name.UNDERFS_OSS_CONNECT_TTL)
           .setDefaultValue(-1)
+          .setDescription("The TTL of OSS connections in ms")
```

Comment:
Do you need the "in ms" part?

Reply:
yes, this is actually an OSS property , which can take -1. If we use `getMs`. I am not sure if it takes -1. so I didn't change it to use `getMs` but leave it as what OSS client requires